docs/redoc/master/openapi.json

```diff
@@ -6136,14 +6136,14 @@
             }
           },
           {
-            "$ref": "#/components/schemas/NamedVector"
+            "$ref": "#/components/schemas/NamedDenseVector"
           },
           {
             "$ref": "#/components/schemas/NamedSparseVector"
           }
         ]
       },
-      "NamedVector": {
```

Comment:
This looks like a breaking change for the clients generation.
This will change the types in the users code.
WDYT?

Reply:
>  This will change the types in the users code.

Correct. I'm afraid we can't do that.

We could make the change internally and do a serde [rename](https://serde.rs/container-attrs.html), though that will make the two inconsistent.

Reply:
+, don't change this

Reply:
But it is not about serde, it is about type names in clients. If we generate clients with this renaming, old code on the client side won't work.

Reply:
Okay, will revert this type. Type rename is unsafe. What about enum variant renaming?

Reply:
> But it is not about serde, it is about type names in clients. If we generate clients with this renaming, old code on the client side won't work.

What I meant was that we can consider to rename these types internally, and utilize serde rename to keep our API specification consistent.

> What about enum variant renaming?

I'm afraid it's the same story. If it results in a change in our API specifications, it's breaking.

Reply:
Reverted type name.
It seems enum variants remanings do not affect opeanpi generated models